### PR TITLE
New instructor view

### LINKED ISSUE #283: instructor_view branch revival
**Please describe what has to be done**
The instructor_view branch is very outdated but seems to have useful code in it still. I'm looking for someone to pull out the relevant code into a new fresh branch based on current code and integrate it there.

### PR DESCRIPTION
**Pull request by chriswoo59**
### Description of change(s), including why:

Created temporary branch new_instructor_view, which is a copy of instructor_view, and merged current frontend into it.
From what I can see, its main purpose was to create an instructor page, which here is just a bunch of links. Each link goes to instructorLiveStats which is a pre-filled poll with a 5 second countdown timer.

### Type of change(s)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Any work to still do or other concerns:

- Once this is looked over and approved, I need to merge this with the original instructor_view to get rid of this extra branch

### Closing issues:

- Closes #283

### Screenshots (if frontend change) of before and after:
instructor_view before

instructor_view after

live_stats before

live_stats after

### CODE REVIEW

**Comment by chriswoo59** [comment]
uh, ignore this for a bit gotta fix these linting errors

**Comment by chriswoo59** [comment]
ok i guess instructor_view didn't have a linting system yet so I fixed the semicolon errors it's had

**Comment by hbh7** [comment]
So I ran through your merge and through the original code, and I think that there's actually nothing we need here. InstructorPolls seems to be replaced and superseded by GroupPolls. InstructorLiveStats seems to be replaced and superseded by PollManager. The "changes" to PollManager are more of reverting changes than moving forward, so I don't think we need that either. 

My new theory is that we actually merged the branch in the past and never deleted it for some reason, or it got re-pushed after the auto deletion after merging, as it seems everything is actually already here. 

TL;DR: It seems we don't actually need to do anything here. Everything we want looks to already be in the main code line.
